.github/labels.md

```diff
@@ -0,0 +1,249 @@
+and the translation process in general.  It maybe helpful to know the language
+of the translation to proofread.
+
+#### Label `component:UI`
+
+The Pull request deals with the game style, the expected user experience and
+graphical APIs used (GiGi and OpenGL).
+
+#### Label `status:cherry-pick for release`
+
+The Pull Request should be applied to the currently open release branch.  Use
+this label sparsely, only for bug fixes, and document a minimal set of commits
+that should be considered by the release manager for cherry-picking.
+Cherry-picking Pull Requests onto release branches should fix bugs, not
+introduce new ones.
+
+#### Label `status:duplicate`
+
+The topic of the Issue is or was already addressed by another Issue.  Add the
+number of the addressing Issue in a comment and close this Issue.  If the
+closed Issue contains new knowledge that is valuable to solve the addressing
+Issue copy it over to the addressing Issue.
+
+#### Label `status:follow up`
+
+The Pull Request discussion showed that the proposed implementation is
+insufficient in some way or the Pull Request in itself is malformed.  Usually
+not required as we prefer rebasing within the same Pull request to update
+the implementation.  When applying this Label reference the new Pull Request
+number in the old one.
+
+#### Label `status:help wanted`
+
+??? I don't think we really need that.  At least I didn't see any Issue were it
+caused someone required to help to magically appear.  This can be handled better
+with `@`-mentions if someones or some groups special knowledge is required.
+
+#### Label `status:invalid`
+
+The Issue doesn't classified as formal valid Issue report or applicable Pull
+Request.  Reasons for this can be manifold:
+
+* The Issue isn't describing a bug but intended behaviour.
+* The Issue opposes the design goals of the game.
+* The issue contains SPAM.
+* The description is too sparse to make sense out of the intent of the Issue
+  and the Reporter is unresponsive or unwilling to give further details.
+* The Feature implemented wasn't discussed previously with the team.
+* The Pull Request is a merge salad à la Chef and the developer is unable or
+  unwilling to fix this.
+* The commits contain intentional written game-braking or offending changes and
+  the developer is unwilling to change those.
+* Multiple Pull Requests compete to address the same topic, but were not chosen
+  to be merged.
+
+Issues labeled as `status:invalid` shouldn't be merged and should be closed.
+
+#### Label `status:merged`
+
+All relevant commits of this Pull Request were merged into the master
+development branch.  This can either happen by merging, cherry-picking or
+rebasing and pushing onto master.  Close the Issue if GitHub didn't close
+it automatically.
+
+#### Label `status:resolved`
+
+The Issue was resolved satisfactorily, either by answering properly or fixing
+the underlying bug.  Close the Issue.
```

Comment:
Here "Issue" actually refers only to github issues, not PRs, so that probably shoud be indicated somehow - maybe by using lowercase "issue"? Uppercase "Issue" is supposed to mean both, issue and PR, as per the definition given above.